runtime/runtime/src/lib.rs

```diff
@@ -1169,7 +1169,7 @@ impl Runtime {
         let mut prefetcher = TriePrefetcher::new_if_enabled(trie.clone());
 
         if let Some(prefetcher) = &mut prefetcher {
-            let _queue_full = prefetcher.input_transactions(transactions);
+            _ = prefetcher.prefetch_transactions_data(transactions);
```

Comment:
With this change, it's slightly harder to understand why we ignore the result.

Optional: add a comment